src/apib-mock/blocks/attributes.apib

```diff
@@ -144,3 +144,11 @@
                     "status"
                 ]
             }
+
++ Response 200 (application/json)
+  Атрибуты с markdown-описанием
+    + Attributes (object)
+        + `bold_text` (string) - текст с **жирным начертанием**
+        + `italic_text` (string) - текст с _курсивным начертанием_
+        + `code_text` (string) - текст с `undefined is not a function`
+        + `link_text` (string) - текст со [ссылкой](https://example.com/)
```

Comment:
It should be noticed that strikethrough formatting (~~like this~~) is not available for us. Commonmark parser does not support it, there is a [discussion](https://talk.commonmark.org/t/strikeout-threw-out-strikethrough-strikes-out-throughout/820) which started almost 10 years ago